website/blog/2017-03-24-krishi-iot-solution.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: blog
+title: Vorto in Krishi IoT Solution
```

Comment:
Can you think of a better  , catchy title where the user gets the content by looking at this title?